src/php/ext/grpc/call.c

```diff
@@ -217,6 +217,12 @@ PHP_METHOD(Call, __construct) {
   }
   wrapped_grpc_channel *channel =
     PHP_GRPC_GET_WRAPPED_OBJECT(wrapped_grpc_channel, channel_obj);
+  if (channel->wrapper == NULL || channel->wrapper->wrapped == NULL) {
```

Comment:
Leave just `channel->wrapper == NULL` seems fine.

Reply:
Just pudated

Reply:
Thanks for fixing it!